config.py

```diff
@@ -0,0 +1,87 @@
+"""
+Configuration settings for the Bucket List API.
+
+The definition of the different configuration settings is contained here:
+- Development Configuration
+- Testing Configuration
+- Production Configuration
+"""
+
+import os
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config:
+    """
+    The definition of the global configuration is defined here.
+
+    Attributes such as SECRET_KEY are the same no matter the platform used.
+    """
+
+    SECRET_KEY = 'u35dfh___vv@$%%jkdsvjkhb___jdfnv93043klm__vdmkll55vd----__'
```

Comment:
I thought you were thinking about using environment variables. That would have been cool.